packages/vue-components/src/Dropdown.vue

```diff
@@ -230,7 +230,10 @@ export default {
     }
 
     .navbar .dropdown-toggle {
-        color: inherit;
```

Comment:
Is there a particular reason why we are removing this property?

Reply:
I think @petermonky removed it because the color inherit will be overridden so it doesn't make a difference